tests/layer_tests/tensorflow_tests/test_tf_MatrixDiag.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
```

Comment:
Will be great to see a link to a documentation in a header.

Reply:
We can consider to add this in the separate PR and update all tests with it.